docs/UserGuide.md

```diff
@@ -239,6 +239,20 @@ Examples:
 * `unmarkAtd 1 t/1` (unmark attendance of student with index 1 for tutorial 1)
 * `unmarkAtd 2 t/12` (unmark attendance of student with index 2 for tutorial 12)
 
+### Mark Group Attendance: `markGroupAtd`
+<a name="markGroupAtd"></a>
+
+Marks the attendance of a group of students for that tutorial.
+
+Format: `markGroupAtd GROUP t/TUTORIAL`
+
+`GROUP`: A string representing the group of students as shown in the list.
+
+`TUTORIAL`: An integer between 1 and 12 (inclusive)
+
+Marking the attendance for a week that is already marked will result in the message
+`This week's attendance has already been marked!`
```

Comment:
Maybe you can consider being lenient regarding this since it's marking the entire tutorial group. However, having this check works fine too!